www/src/utils/node/creators.js

```diff
@@ -0,0 +1,69 @@
+const Promise = require(`bluebird`)
+const path = require(`path`)
+const slash = require(`slash`)
+const slugify = require(`slugify`)
+
+exports.createPages = ({ graphql, actions }) => {
+  const { createPage } = actions
+
+  return new Promise((resolve, reject) => {
```

Comment:
Could we use [async/await syntax](https://github.com/gatsbyjs/gatsby/pull/21210/files#diff-fb043578f52339bcb2a2d2b4df1203c8R112) for these? While we're refactoring, we might as well pull everything up to date with modern syntax :)

Reply:
Just read your TODO item -- we don't have to wait. You can just convert it to async here, and we can close out my PR.